ProgramFlow/ProgramFlow/Flow.cs

```diff
@@ -29,8 +29,23 @@ public static bool InvertBoolean(bool value)
         /// <returns>The correct <see cref="DriverAction"/> value.</returns>
         public static DriverAction DriveSafelyIfElse(LightColor color)
         {
+            string lightColor = color.ToString();
+
             // TODO: Fix this method so it passes all associated tests. An if-else statement must be used to return the correct values.
-            return (DriverAction)24;
+            if (lightColor == "Green")
```

Comment:
Rather than doing a string comparison, it would be preferable to compare to the appropriate enum value, such as:

if (color == LightColor.Green)

It is more efficient and it allows for refactoring of enum values overtime without risking breaks that will only occur at runtime.